### PR TITLE
Test coverage for LRO Parametrized endpoints

### PR DESCRIPTION
**Pull request by joheredi**
* Adding test coverage for LRO Parametrized endpoints scenarios.
* Updating core-http and enabling test for `arrayStringMultiEmpty`

### CODE REVIEW

File: test/integration/lroParametrizedEndpoints.spec.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  beforeEach(() => {
+    client = new LroParametrizedEndpointsClient({ host: "host:3000" });
+  });
+
+  it("should pollWithParameterizedEndpoints", async () => {
+    const poller = await client.pollWithParameterizedEndpoints("local");
+    poller.delay = () => Promise.resolve();
+    const result = await poller.pollUntilDone();
+    assert.equal(result.body, "success");
+  }).timeout(10000);
```
Comment (sarangan12) [inline test/integration/lroParametrizedEndpoints.spec.ts:16]: Do we need this timeout?

**Review by sarangan12 — APPROVED** [review]
Left one minor comment. Otherwise code looks fine

File: test/integration/lroParametrizedEndpoints.spec.ts (hunk 1 above)
Comment (joheredi) [reply test/integration/lroParametrizedEndpoints.spec.ts:16]: We don't! good catch

**Review by deyaaeldeen — COMMENTED** [review]
Looks good, thanks for doing it!

**Pull request merged**